packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -401,17 +401,15 @@ export class AtomicBreadbox implements InitializableComponent {
 
     return (
       <div class="text-on-background text-sm flex">
-        <span part="label" class="font-bold p-2 pl-0">
+        <span part="label" class="font-bold py-[0.625rem] pl-0 pr-2">
```

Comment:
This fix isn't strongly related to the PR. The alignment for the "Filters:" text before the breadbox was off-center and I fixed it.